### PR TITLE
[do not merge] Rebasing feature/AzureCommunication onto master

### PR DESCRIPTION
**Pull request by knvsl**
Rebase feature/AzureCommunication onto master, will need to force push to feature branch. This is to review that the merge back to master after rebasing looks ok.

- Renamed CommunicationAuthenticationPolicy -> CommunicationPolicyTokenCredential
- Renamed CommunicationUserCredential -> CommunicationTokenCredential in Chat and some of the tests

### CODE REVIEW

File: sdk/communication/AzureCommunication/Tests/CommunicationPolicyTokenCredentialTests.swift [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            if let error = error {
+                XCTFail("Create token timed out: \(error)")
+            }
+        }
+    }
+    
+    func testCreateRefreshableWithInitialToken() throws {
+        let expectation = self.expectation(description: "Create refreshable with initial token")
+        let token = expiredToken
+        let options = CommunicationTokenRefreshOptions(
```
Comment (knvsl) [inline sdk/communication/AzureCommunication/Tests/CommunicationPolicyTokenCredentialTests.swift:90]: @JoshuaLai If you can review these tests still look ok, I just had to make a couple updates to use the CommunicationTokenRefreshOptions

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,34 +4,8 @@
 
 ### Breaking Changes
 - Azure Communication Common Library
- - Communication identifier `MicrosoftTeamsUserIdentifier` property `identifier` renamed to `userId` since identifier was too generic.
- 
-## 1.0.0-beta.7 (2021-01-12)
```
Comment (JoshuaLai) [inline CHANGELOG.md:9]: I think we should keep both changes here. 
You are missing the `1.0.0-beta.7 (2021-01-12)` change set

File: sdk/communication/AzureCommunication/Tests/CommunicationPolicyTokenCredentialTests.swift (hunk 1 above)
Comment (JoshuaLai) [reply sdk/communication/AzureCommunication/Tests/CommunicationPolicyTokenCredentialTests.swift:90]: Yes everything still looks good

File: CHANGELOG.md (hunk 2 above)
Comment (knvsl) [reply CHANGELOG.md:9]: thanks!! updated

**Pull request closed without merging**